<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
  
  

  <head>
    <title>SomaticSniper (v1.0.5.0)</title>
    <link rel="shortcut icon" href="/gmt_favicon.png" />
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="content-language" content="en" />
<link rel="stylesheet" href="/res/css/reset.css?20150720140317" media="screen,print"/>
<link rel="stylesheet" href="/res/css/main_sub.css?20150720140317" media="screen,print"/>

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="/res/js/jquery.currentPage.js"></script>
<script type="text/javascript" src="/res/js/jquery-ui.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
        $("#menu_main a").currentPage({
            indexPage: "index.html",
            anyPath: true
        });
    });
</script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24132169-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


  </head>
  <body>
    <div id="container">
      <div id="header">
          <div id="menu_top">
              <a href="/index.html" alt="GMT Main Site" id="gmt_link"> </a>
          </div>

          <div id="title">
              <div id="icon">
                  
                    <img src="/packages/somatic-sniper/res/images/gmt_somatic_sniper_48.png" width="48" height="48" alt="SomaticSniper"/>
                  
              </div>
              <div id="site_title">
                  <h1>SomaticSniper <span class="version">v1.0.5.0</span></h1>
              </div>
          </div>

          <div id="menu_main">
              <ul>
                  
                  <li><a href="index.html">Overview</a></li>
                  
                  <li><a href="install.html">Installation</a></li>
                  
                  <li><a href="documentation.html">Documentation</a></li>
                  
              </ul>
          </div>
      </div>
      <div id="content">

        <div id="helix_bottom">
          <img src="/res/images/helix_bottom.png" width="230" height="49" alt="helix_bottom" />
        </div>

        <div id="content_main">

          <h2></h2>
          <div class="rounded_bg">
              <h2 id="somaticsniper-user-manual">SomaticSniper User Manual</h2>

<p><strong>David E. Larson, Travis E. Abbott and Christopher C. Harris</strong></p>

<p><strong>October 26, 2011</strong></p>

<p>The purpose of this program is to identify single nucleotide positions that are different between tumor and normal (or, in theory, any two bam files). It takes a tumor bam and a normal bam and compares the two to determine the differences. It outputs a file in a format very similar to Samtools consensus format. It uses the genotype likelihood model of MAQ (as implemented in Samtools) and then calculates the probability that the tumor and normal genotypes are different. This probability is reported as a somatic score. The somatic score is the Phred-scaled probability (between 0 to 255) that the Tumor and Normal genotypes are not different where 0 means there is no probability that the genotypes are different and 255 means there is a probability of 1 − 10^(255∕−10) that the genotypes are different between tumor and normal. This is consistent with how the SAM format reports such probabilities.</p>

<p>There are two modes, the joint genotyping mode (-J) takes into account the fact that the tumor and normal samples are not entirely independent and also takes into account the prior probability of a somatic mutation. This probability can be scaled to control the sensitivity of the algorithm. An accurate value for this prior would be 0.000001, but this may result in a severe lack of sensitivity at lower depths. A less realistic prior probability will generate more sensitive results at the expense of an increase in the number of false positives. To get a similar sensitivity to the default mode, we recommend using a prior of 0.01. The default mode treats the two samples as if they came from two different individuals. This mode uses a less accurate mathematical model, but yields good results, especially if the normal may contain some tumor cells or the tumor is quite impure.</p>

<h2 id="usage">Usage</h2>

<p><code>bam-somaticsniper [options] -f &lt;ref.fasta&gt; &lt;tumor.bam&gt; &lt;normal.bam&gt; &lt;snv_output_file&gt;</code></p>

<p><strong>Required Option:</strong></p>

<p><code>-f   FILE REQUIRED reference sequence in the FASTA format</code></p>

<p><strong>Options:</strong></p>

<p><code>-q  INT filtering reads with mapping quality less than INT [0]</code></p>

<p><code>-Q  INT filtering somatic snv output with somatic quality less than INT [15]</code></p>

<p><code>-L FLAG do not report LOH variants as determined by genotypes</code></p>

<p><code>-G FLAG do not report Gain of Referene variants as determined by genotypes</code></p>

<p><code>-p  FLAG disable priors in the somatic calculation. Increases sensitivity for solid tumors.</code></p>

<p><code>-J  FLAG Use prior probabilities accounting for the somatic mutation rate</code></p>

<p><code>-s  FLOAT prior probability of a somatic mutation (implies -J) [0.01]</code></p>

<p><code>-T  FLOAT theta in maq consensus calling model (for -c/-g) [0.850000]</code></p>

<p><code>-N  INT number of haplotypes in the sample (for -c/-g) [2]</code></p>

<p><code>-r  FLOAT prior of a diﬀerence between two haplotypes (for -c/-g) [0.001000]</code></p>

<p><code>-F  STRING select output format (vcf or classic) [classic]</code></p>

<h3 id="notes-on-running-somaticsniper">Notes on running SomaticSniper</h3>

<p>Minimally, you must provide the program the reference fasta the bams were aligned against (passed with the -f option), a tumor bam, a normal bam, and the filename of the resulting output file. We recommend filtering out reads with a mapping quality of 0 (i.e. use -q 1) as they are typically randomly placed in the genome. We have also found that few variants with a somatic score less than 15 validate, but you may decrease the minimum score or increase it to a higher threshold (eg -Q 40). To obtain high confidence sites, we recommend also thresholding the minimum average mapping quality for the variant base to 40 for reads aligned with BWA or 70 for reads aligned with MAQ. We have not tested other aligners at this time. Disabling priors is not recommended, but may increase sensitivity at the cost of a decrease in specificity.</p>

<h4 id="current-recommended-settings">Current Recommended Settings</h4>

<p>We recommend that you utilize both the <code>-G</code> and <code>-L</code> options when running in order to reduce likely false positives with little impact on sensitivity. An example command-line is below:</p>

<p><code>
bam-somaticsniper -Q 40 -G -L -f reference.fa tumor.bam normal.bam output.txt
</code></p>

<h4 id="basic-filtering-with-provided-perl-scripts">Basic filtering with provided Perl scripts</h4>

<p>A small number of basic Perl scripts are included in the SomaticSniper package (located in src/scripts of the source code release) to aid in filtering out likely false positives. In order to get the recommended filtering you should do the following. Defaults are set assuming that BWA short is the aligner used. Other aligners have not been tested and recommendations are not available. Before proceeding you will need to obtain and compile bam-readcount (https://github.com/genome/bam-readcount). You will also need to generate a samtools pileup (not mpileup) indel file. Handling of indel containing VCFs is not implemented.</p>

<ol>
  <li>
    <p>Filter on standard filters using the indel file. This will also remove LOH calls e.g. <code>perl snpfilter.pl –snp-file your_sniper_file –indel-file your_indel_pileup</code></p>
  </li>
  <li>
    <p>Adapt the remainder for use with bam-readcount e.g. <code>perl prepare_for_readcount.pl –snp-file your_sniper_file.SNPfilter</code></p>
  </li>
  <li>
    <p>Run bam-readcount (I’d recommend using the same mapping quality -q setting as you ran SomaticSniper with) e.g. <code>bam-readcount -b 15 -f your_ref.fasta -l your_sniper_file.SNPfilter.pos your_tumor.bam &gt; your_readcounts.rc</code>
 Run the false positive filter e.g. perl fpfilter.pl –snp-file your_sniper_file.SNPfilter –readcount-file your_readcounts.rc</p>
  </li>
  <li>
    <p>Lastly, run the “high confidence” filter which filters based on the Somatic Score and mapping quality e.g. <code>perl highconfidence.pl –snp-file your_sniper_file.SNPfilter.fp_pass</code></p>
  </li>
</ol>

<p>Your final set of high confidence and highly filtered indels is now in the file <code>your_sniper_file.SNPfilter.fp_pass.hc</code></p>

<h3 id="file-formats">File Formats</h3>

<p>The output by SomaticSniper consists of line for all sites whose consensus differs from the reference base. Each of the three available output formats is described below</p>

<p><strong>Classic:</strong></p>

<p>Each line contains the following tab-separated values:</p>

<ol>
  <li>Chromosome</li>
  <li>Position</li>
  <li>Reference base</li>
  <li>IUB genotype of tumor</li>
  <li>IUB genotype of normal</li>
  <li>Somatic Score</li>
  <li>Tumor Consensus quality</li>
  <li>Tumor variant allele quality</li>
  <li>Tumor mean mapping quality</li>
  <li>Normal Consensus quality</li>
  <li>Normal variant allele quality</li>
  <li>Normal mean mapping quality</li>
  <li>Depth in tumor (# of reads crossing the position)</li>
  <li>Depth in normal (# of reads crossing the position)</li>
  <li>Mean base quality of reads supporting reference in tumor</li>
  <li>Mean mapping quality of reads supporting reference in tumor</li>
  <li>Depth of reads supporting reference in tumor</li>
  <li>Mean base quality of reads supporting variant(s) in tumor</li>
  <li>Mean mapping quality of reads supporting variant(s) in tumor</li>
  <li>Depth of reads supporting variant(s) in tumor</li>
  <li>Mean base quality of reads supporting reference in normal</li>
  <li>Mean mapping quality of reads supporting reference in normal</li>
  <li>Depth of reads supporting reference in normal</li>
  <li>Mean base quality of reads supporting variant(s) in normal</li>
  <li>Mean mapping quality of reads supporting variant(s) in normal</li>
  <li>Depth of reads supporting variant(s) in normal</li>
</ol>

<p><strong>VCF</strong></p>

<p>VCF output from SomaticSniper conforms to version 4.1 of the VCF specification. Hence, each non-header output line contains the following fields:</p>

<ol>
  <li>Chromosome</li>
  <li>Position</li>
  <li>ID (unused)</li>
  <li>Reference base</li>
  <li>Alternate bases (comma separated)</li>
  <li>Quality (unused)</li>
  <li>Filters (unused)</li>
  <li>INFO (unused)</li>
  <li>FORMAT specification for each sample</li>
  <li>NORMAL sample data</li>
  <li>TUMOR sample data</li>
</ol>

<p>The following FORMAT fields will be populated for each of NORMAL and TUMOR.</p>

<table>
  <thead>
    <tr>
      <th>ID</th>
      <th>Number</th>
      <th>Type</th>
      <th>Description</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>GT</td>
      <td>1</td>
      <td>String</td>
      <td>Genotype</td>
    </tr>
    <tr>
      <td>IGT</td>
      <td>1</td>
      <td>String</td>
      <td>Genotype when called independently (only filled if called in joint prior mode)</td>
    </tr>
    <tr>
      <td>DP</td>
      <td>1</td>
      <td>Integer</td>
      <td>Total read depth</td>
    </tr>
    <tr>
      <td>DP4</td>
      <td>4</td>
      <td>Integer</td>
      <td>Number of high-quality ref-forward bases, ref-reverse, alt-forward and alt-reverse bases</td>
    </tr>
    <tr>
      <td>BCOUNT</td>
      <td>4</td>
      <td>Integer</td>
      <td>Occurrence count for each base at this site (A,C,G,T)</td>
    </tr>
    <tr>
      <td>GQ</td>
      <td>1</td>
      <td>Integer</td>
      <td>Genotype quality</td>
    </tr>
    <tr>
      <td>JGQ</td>
      <td>1</td>
      <td>Integer</td>
      <td>Joint genotype quality (only filled if called in joint prior mode)</td>
    </tr>
    <tr>
      <td>VAQ</td>
      <td>1</td>
      <td>Integer</td>
      <td>Variant quality</td>
    </tr>
    <tr>
      <td>BQ</td>
      <td>.</td>
      <td>Integer</td>
      <td>Average base quality of each base in the call, reported in alphabetical order (A,C,G,T)</td>
    </tr>
    <tr>
      <td>MQ</td>
      <td>1</td>
      <td>Integer</td>
      <td>Average mapping quality across all reads.</td>
    </tr>
    <tr>
      <td>AMQ</td>
      <td>.</td>
      <td>Integer</td>
      <td>Average mapping quality of each base in the call, reported in alphabetical order (A,C,G,T)</td>
    </tr>
    <tr>
      <td>SS</td>
      <td>1</td>
      <td>Integer</td>
      <td>Variant status relative to non-adjacent normal: 0=wildtype, 1=germline, 2=somatic, 3=LOH, 4=unknown</td>
    </tr>
    <tr>
      <td>SSC</td>
      <td>1</td>
      <td>Integer</td>
      <td>Somatic Score</td>
    </tr>
  </tbody>
</table>

<h3 id="user-support">User Support</h3>

<p>Please first search <a href="http://www.biostars.org">Biostar</a> and then ask a question there if needed. We automatically monitor Biostar for questions related to our tools. </p>



          </div>
            
            
        </div>

        <div id="content_sidebar">
          
  <div id="rounded_bg">
    <div align="center">
      <a href="install.html" class="btn_install">Install</a> <br/>

      
        <a href="https://github.com/genome/somatic-sniper" class="btn_view_on_github">View on GitHub</a> <br/>
      

      
        <a href="http://biostars.org/t/SomaticSniper" class="btn_get_help">Get Help</a> <br />
      
    </div>
  </div>
  <br/>



  <div class="sidebar_menu">
    <h3>Versions</h3>
    <hr/>
    <ul>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.5.0">v1.0.5.0</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.4.2">v1.0.4.2</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.4">v1.0.4</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.3">v1.0.3</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.2.3">v1.0.2.3</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.1">v1.0.1</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.0.2">v1.0.0.2</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v0.7.4.9">v0.7.4.9</a>
      </li>
    
    </ul>
  </div>
  <br/>





        </div>

      </div>

      <div id="footer">
        <p>Genome Modeling Tools is produced by <a href="http://genome.wustl.edu/">The Genome Institute at Washington University School of Medicine</a></p>

      </div>
    </div>
  </body>
</html>
