<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
  
  

  <head>
    <title>SomaticSniper (v1.0.4)</title>
    <link rel="shortcut icon" href="/gmt_favicon.png" />
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="content-language" content="en" />
<link rel="stylesheet" href="/res/css/reset.css?20140813154312" media="screen,print"/>
<link rel="stylesheet" href="/res/css/main_sub.css?20140813154312" media="screen,print"/>

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="/res/js/jquery.currentPage.js"></script>
<script type="text/javascript" src="/res/js/jquery-ui.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
        $("#menu_main a").currentPage({
            indexPage: "index.html",
            anyPath: true
        });
    });
</script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24132169-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


  </head>
  <body>
    <div id="container">
      <div id="header">
          <div id="menu_top">
              <a href="/index.html" alt="GMT Main Site" id="gmt_link"> </a>
          </div>

          <div id="title">
              <div id="icon">
                  
                    <img src="/packages/somatic-sniper/res/images/gmt_somatic_sniper_48.png" width="48" height="48" alt="SomaticSniper"/>
                  
              </div>
              <div id="site_title">
                  <h1>SomaticSniper <span class="version">v1.0.4</span></h1>
              </div>
          </div>

          <div id="menu_main">
              <ul>
                  
                  <li><a href="index.html">Overview</a></li>
                  
                  <li><a href="install.html">Installation</a></li>
                  
                  <li><a href="documentation.html">Documentation</a></li>
                  
              </ul>
          </div>
      </div>
      <div id="content">

        <div id="helix_bottom">
          <img src="/res/images/helix_bottom.png" width="230" height="49" alt="helix_bottom" />
        </div>

        <div id="content_main">

          <h2></h2>
          <div class="rounded_bg">
              <h2>Introduction</h2>
<hr />

<p>The purpose of this program is to identify single nucleotide positions that are different between tumor and normal (or, in theory, any two bam files). It takes a tumor bam and a normal bam and compares the two to determine the differences. It outputs a file in a format very similar to Samtools consensus format. It uses the genotype likelihood model of MAQ (as implemented in Samtools) and then calculates the probability that the tumor and normal genotypes are different. This probability is reported as a somatic score. The somatic score is the Phred-scaled probability (between 0 to 255) that the Tumor and Normal genotypes are not different where 0 means there is no probability that the genotypes are different and 255 means there is a probability of 1 – 10<sup>(255/-10)</sup> that the genotypes are different between tumor and normal. This is consistent with how the SAM format reports such probabilities. It is currently available as source code via github or as a Debian APT package.
</p>

<p>David E. Larson, Travis E. Abbott and Christopher C. Harris
October 26, 2011
</p>

<h2>How to Cite</h2>
<hr />

<p>Citations for SomaticSniper should reference the <a href="http://bioinformatics.oxfordjournals.org/content/28/3/311">Bioinformatics</a> paper.</p>



          </div>
            
            
            <h2 style="margin: 1em 20px">Latest SomaticSniper News:</h2>
            <hr/>
            
            
            
            
            

<div class="post rounded_bg">
  <h2 class="entry-title"><a href="/2014/06/14/somatic-sniper_v1.0.4_released.html">SomaticSniper v1.0.4</a></h2>
  <div class="entry-date"><abbr title="June 14, 2014" class="published">June 14, 2014</abbr></div>
  <div class="entry-content">
    
      <div class="post_icon">
          <div class="project_icon"><img src="/packages/somatic-sniper/res/images/gmt_somatic_sniper_48.png" /></div>
      </div>
    

    <h4 id="major-changes">Major Changes</h4>
<p>This release adds options to filter loss-of-heterozygosity and gain-of-reference allele calls from SomaticSniper output. Loss-of-heterozygosity calls are defined as calls where the tumor genotype is fully a subset of the normal genotype. Gain-of-heterozygosity calls are defined as those where the reference allele is not present in the normal genotype call, but <em>is</em> present in the tumor. For example the following calls would both be suppressed,</p>

<table>
  <thead>
    <tr>
      <th>Ref</th>
      <th>Tumor</th>
      <th>Normal</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>A</td>
      <td>GG</td>
      <td>AG</td>
    </tr>
    <tr>
      <td>A</td>
      <td>AG</td>
      <td>GG</td>
    </tr>
  </tbody>
</table>

<h4 id="minor-changes">Minor Changes</h4>
<ul>
  <li>build-common is now utilized as a git subtree. Downloads from github will now be functional and recursive clones are no longer necessary. </li>
</ul>


<p>You can <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.4">download this release on Github.</a></p>
  </div>

</div>

            
            
            
            
            
            
            
            
            
            
            
            
            
            
            
            
            
            
            
            

<div class="post rounded_bg">
  <h2 class="entry-title"><a href="/2012/05/08/somatic-sniper_v1.0.0.2_released.html">SomaticSniper v1.0.0</a></h2>
  <div class="entry-date"><abbr title="May 08, 2012" class="published">May 08, 2012</abbr></div>
  <div class="entry-content">
    
      <div class="post_icon">
          <div class="project_icon"><img src="/packages/somatic-sniper/res/images/gmt_somatic_sniper_48.png" /></div>
      </div>
    

    <p>The latest release of SomaticSniper, The Genome Institute’s somatic
SNV calling workhorse, adds an alternative statistical model that
better accounts for the rarity of somatic events by jointly
considering the tumor and normal genotypes.  This version also adds
native support for the VCF and BED formats as output.  The VCF output
contains information useful for downstream filtering, e. g., fraction
of reads on the forward and reverse strands, average read mapping
quality, and average base quality for reads/bases supporting the
variant allele and those supporting the reference allele.</p>


<p>You can <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.0.2">download this release on Github.</a></p>
  </div>

</div>

            
            
            
            
            
            
            
            

<div class="post rounded_bg">
  <h2 class="entry-title"><a href="/2011/07/12/somatic-sniper_v0.7.4.9_released.html">SomaticSniper v0.7.4 Released</a></h2>
  <div class="entry-date"><abbr title="July 12, 2011" class="published">July 12, 2011</abbr></div>
  <div class="entry-content">
    
      <div class="post_icon">
          <div class="project_icon"><img src="/packages/somatic-sniper/res/images/gmt_somatic_sniper_48.png" /></div>
      </div>
    

    <p>The purpose of this program is to identify single nucleotide positions that are different between tumor and normal (or, in theory, any two bam files). It takes a tumor bam and a normal bam and compares the two to determine the differences. It outputs a file in a format very similar to Samtools consensus format. It uses the genotype likelihood model of MAQ (as implemented in Samtools) and then calculates the probability that the tumor and normal genotypes are different. This probability is reported as a somatic score. The somatic score is the Phred-scaled probability (between 0 to 255) that the Tumor and Normal genotypes are not different where 0 means there is no probability that the genotypes are different and 255 means there is a probability of 1 – 10<sup>(255/-10)</sup> that the genotypes are different between tumor and normal. This is consistent with how the SAM format reports such probabilities.</p>


<p>You can <a href="https://github.com/genome/somatic-sniper/releases/tag/v0.7.4.9">download this release on Github.</a></p>
  </div>

</div>

            
            
            
            
            
            
            
            
            
        </div>

        <div id="content_sidebar">
          
  <div id="rounded_bg">
    <div align="center">
      <a href="install.html" class="btn_install">Install</a> <br/>

      
        <a href="https://github.com/genome/somatic-sniper" class="btn_view_on_github">View on GitHub</a> <br/>
      

      
        <a href="http://biostars.org/t/SomaticSniper" class="btn_get_help">Get Help</a> <br />
      
    </div>
  </div>
  <br/>



  <div class="sidebar_menu">
    <h3>Versions</h3>
    <hr/>
    <ul>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.4">v1.0.4</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.3">v1.0.3</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.2.3">v1.0.2.3</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.1">v1.0.1</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v1.0.0.2">v1.0.0.2</a>
      </li>
    
      <li>
      <a href="https://github.com/genome/somatic-sniper/releases/tag/v0.7.4.9">v0.7.4.9</a>
      </li>
    
    </ul>
  </div>
  <br/>





        </div>

      </div>

      <div id="footer">
        <p>Genome Modeling Tools is produced by <a href="http://genome.wustl.edu/">The Genome Institute at Washington University School of Medicine</a></p>

      </div>
    </div>
  </body>
</html>
