<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
  
  

  <head>
    <title>BreakDancer ()</title>
    <link rel="shortcut icon" href="/gmt_favicon.png" />
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="content-language" content="en" />
<link rel="stylesheet" href="/res/css/reset.css?20140814163756" media="screen,print"/>
<link rel="stylesheet" href="/res/css/main_sub.css?20140814163756" media="screen,print"/>

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="/res/js/jquery.currentPage.js"></script>
<script type="text/javascript" src="/res/js/jquery-ui.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
        $("#menu_main a").currentPage({
            indexPage: "index.html",
            anyPath: true
        });
    });
</script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24132169-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


  </head>
  <body>
    <div id="container">
      <div id="header">
          <div id="menu_top">
              <a href="/index.html" alt="GMT Main Site" id="gmt_link"> </a>
          </div>

          <div id="title">
              <div id="icon">
                  
                    <img src="/packages/breakdancer/res/images/genome_breakdancer_48.png" width="48" height="48" alt="BreakDancer"/>
                  
              </div>
              <div id="site_title">
                  <h1>BreakDancer <span class="version"></span></h1>
              </div>
          </div>

          <div id="menu_main">
              <ul>
                  
                  <li><a href="index.html">Overview</a></li>
                  
                  <li><a href="documentation.html">Documentation</a></li>
                  
                  <li><a href="install.html">Install</a></li>
                  
              </ul>
          </div>
      </div>
      <div id="content">

        <div id="helix_bottom">
          <img src="/res/images/helix_bottom.png" width="230" height="49" alt="helix_bottom" />
        </div>

        <div id="content_main">

          <h2></h2>
          <div class="rounded_bg">
              <h1 id="running-breakdancer">Running BreakDancer</h1>

<hr />

<p>To run BreakDancer, first use bam2cfg.pl to prepare the required per-invocation config file. See the below for parameter descriptions.</p>

<p class="terminal">
/usr/lib/breakdancer-max1.2/bam2cfg.pl bam_files breakdancer_options
</p>
<p>Then run BreakDancer on the config</p>

<p class="terminal">
breakdancer-max config_file.cfg
</p>

<h1 id="example-pipeline">EXAMPLE PIPELINE</h1>

<hr />

<p>Please see below for detailed information about the <em>bam2cfg.pl</em> and <em>breakdancer-max</em> commands.</p>

<h2 id="step-1">STEP 1</h2>
<p>Create a configuration file using bam2cfg.pl:</p>

<p>The precompiled Debian package will install this in /usr/lib/breakdancer-max along with a few required Perl modules.</p>

<p class="terminal">
/usr/lib/breakdancer-max/bam2cfg.pl -g -h tumor.bam normal.bam &gt; BRC6.cfg
</p>

<h2 id="step-2">STEP 2</h2>
<p>Detect inter-chromosomal translocations:</p>

<p class="terminal">
breakdancer_max -t -q 10 -d BRC6.ctx BRC6.cfg &gt; BRC6.ctx
</p>

<p>The -d option dumps CTX supporting read pairs into fastq files (in this case BRC6.ctx) by library.</p>

<p>This step normally takes 12 hours or so for three bam files, 8 hours or so for two bam files for cpp version, around three days for perl version.</p>

<h1 id="bam2cfg">bam2cfg</h1>

<hr />

<h2 id="name">NAME</h2>
<p>bam2cfg - create a configuration file for BreakDancer</p>

<h2 id="synopsis">SYNOPSIS</h2>
<pre><code>bam2cfg.pl [options] input_bams &gt; config_file
</code></pre>

<h2 id="notes">NOTES</h2>
<p>Manually view the insert size and flag distribution results in the output .cfg file to see if there are any data quality issue. Usually std/mean should be &lt; 0.2 or 0.3 at most. The flag 32(x%), represents percent of chimeric insert, this number (x%) should usually be smaller than 3%.</p>

<p>View png files for the insert size distribution. You should usually see a normal distribution, a bimodal distribution is undesirable and it is not recommended to continue BreakDancerMax step with this situation existing.</p>

<h2 id="options">OPTIONS</h2>
<dl>
<dt>-q INT</dt>
<dd>Minimum mapping quality [default = 35]</dd>

<dt>-m</dt>
<dd>Using mapping quality instead of alternative mapping quality</dd>

<dt>-s</dt>
<dd>Minimal mean insert size [default = 50]</dd>

<dt>-C</dt>
<dd>Change default system from Illumina to SOLiD</dd>

<dt>-c FLOAT</dt>
<dd>Cut off in unit of standard deviation [default = 4]</dd>

<dt>-n INT</dt>
<dd>Number of observation required to estimate mean and s.d. insert size [10000]</dd>

<dt>-v FLOAT</dt>
<dd>Cutoff on coefficients of variation [default = 1]</dd>

<dt>-f STRING</dt>
<dd>A two column tab-delimited text file (RG, LIB) specify the RG=&gt;LIB mapping, useful when BAM header is incomplete</dd>

<dt>-b INT</dt>
<dd>Number of bins in the histogram [default = 50]</dd>

<dt>-g</dt>
<dd>Output mapping flag distribution</dd>

<dt>-h</dt>
<dd>Plot insert size histogram for each BAM library</dd>
</dl>

<h1 id="breakdancermax">BreakDancerMax</h1>

<hr />

<h2 id="name-1">NAME</h2>
<p>BreakDancerMax - SV detection</p>

<h2 id="synopsis-1">SYNOPSIS</h2>

<p>breakdancer-max [options] config_file</p>

<h2 id="options-1">OPTIONS</h2>
<dl>
<dt>-o STRING</dt>
<dd>operate on a single chromosome [default is all chromosomes]</dd>

<dt>-c INT</dt>
<dd>cutoff in unit of standard deviation [default = 3]</dd>

<dt>-t</dt>
<dd>only detect transchromosomal rearrangement</dd>

<dt>-p FLOAT</dt>
<dd>prior probability of SV [default = 0.001] (Not available in C++ version)</dd>

<dt>-e</dt>
<dd>learn parameters from data before applying to SV detection (Not available in C++ version)</dd>

<dt>-l</dt>
<dd>analyze Illumina long insert (mate-pair) library</dd>

<dt>-f</dt>
<dd>use Fisher's method to combine P values from multiple library (Not available in C++ version)</dd>

<dt>-d STRING</dt>
<dd>prefix of fastq files that SV supporting reads will be saved by library</dd>

<dt>-g _FILE_</dt>
<dd>dump SVs and supporting reads in BED format for GBrowse</dd>

<dt>-m INT</dt>
<dd>maximum SV size [default = 1000000]</dd>

<dt>-q INT</dt>
<dd>minimum alternative mapping quality [default = 35]</dd>

<dt>-s INT</dt>
<dd>minimum length of a region [default = 7]</dd>

<dt>-b INT</dt>
<dd>buffer size for building connection [default = 100]</dd>

<dt>-r INT</dt>
<dd>minimum number of read pairs required to establish a connection</dd>

<dt>-x INT</dt>
<dd>maximum threshold of haploid sequence coverage for regions to be ignored [default = 1000]</dd>

<dt>-a</dt>
<dd>print out copy number by bam file rather than library [defaults on]</dd>

<dt>-h</dt>
<dd>print out Allele Frequency column [defaults off]</dd>

<dt>-y INT</dt>
<dd>output score filter [default = 30]</dd>
</dl>

<h2 id="description">DESCRIPTION</h2>
<p>Most of these options are self-explanatory. It is convenient to use the -o option to parallelize SV detection for each chromosome. When -o is used, the detection of inter-chromosomal translocation is disabled. In that case, it may be convenient to use -t in a separate process to detect putative inter-chromosomal translocations without bothering to analyze read pairs that are mapped to the same chromosome.</p>

<p>BreakDancer only supports properly formatted bam files and has only been tested using bam files produced by BWA. To obtain the correct result, it is important to have readgroup (@RG) tag in both the header and each alignment in the bam files. </p>

<p>The input to breakdancer-max is a set of map files produced by a front-end aligner such as MAQ, BWA, NovoAlign and Bfast, and a tab-delimited configuration file that specifies the locations of the map files, the detection parameters, and the sample information.</p>

<h3 id="configuration">CONFIGURATION</h3>
<p>If your map files are in the sam/bam format, you can use the bam2cfg.pl in the released package to automatic generate a configuration file (bam2cfg.pl also has dependence on AlnParser.pm in the release package). If you have a single bam file that contains multiple libraries, make sure that the readgroup and library information are properly encoded in the sam/bam header, and in each alignment record, otherwise bam2cfg.pl may fail to produce a correct configuration file. Please follow instructions on <a href="http://samtools.sourceforge.net">SourceForge</a> to properly format your bam files.</p>

<p>An example manual configuration file is like this </p>

<p class="terminal">
map:1.map mean:219 std:18 readlen:36.00 sample:tA exe:maq-0.6.8 mapview -b 
map:2.map mean:220 std:19 readlen:36.00 sample:tB exe:maq-0.6.8 mapview -b 
map:3.map mean:219 std:18 readlen:36.00 sample:nA exe:maq-0.7.1 mapview -b 
map:4.map mean:219 std:18 readlen:36.00 sample:nB exe:maq-0.7.1 mapview -b 
</p>

<p>An example configuration file produced by bam2cfg.pl look like this: </p>

<p class="terminal">
readgroup:2825107881 platform:illumina map:tumor.bam readlen:75.00 lib:demolib1
num:10001 lower:86.83 upper:443.91 mean:315.09 std:43.92 exe:samtools view 
readgroup:2843249908 platform:illumina map:tumor.bam readlen:75.00 lib:demolib1
num:10001 lower:86.83 upper:443.91 mean:315.09 std:43.92 exe:samtools view 
readgroup:2843255910 platform:illumina map:normal.bam readlen:75.00 lib:demolib2
num:10001 lower:95.36 upper:443.31 mean:311.68 std:42.86 exe:samtools view 
readgroup:2843255906 platform:illumina map:normal.bam readlen:75.00 lib:demolib2
num:10001 lower:95.36 upper:443.31 mean:311.68 std:42.86 exe:samtools view 
</p>

<p>Each row must contain at least 6 key:value pairs (separated by colon) that specify:</p>

<ol>
  <li>the location of the map file</li>
  <li>the mean insert size</li>
  <li>the standard deviation insert size</li>
  <li>the average read length</li>
  <li>a unique identifier assigned to the map file (usually representing a PE library)</li>
  <li>a command line that can run by perl system calls to produce MAQ mapview alignment</li>
</ol>

<p>Listing multiple map files in a single configuration file would automatically enable pooled analysis: reads from all the map files are jointly analyzed to find unified SV hypotheses across all the map files.</p>

<h3 id="separation-thresholds">SEPARATION THRESHOLDS</h3>
<p>In addition to the above 6 keys: map, mean, std, readlen, sample, and exe, BreakDancerMax allows users to explicitly specify the separation thresholds using the keys: upper and lower. For example:</p>

<p class="terminal">
map:1.map upper:300 lower:100 readlen:36.00 sample:tA exe:maq-0.6.8 mapview -b
</p>

<p>This will instruct BreakDancerMax to detect deletions using read pairs that are at least 300 bp apart (outer distance) and detect insertions using read pairs that are at most 100 bp apart.</p>

<p>The upper and the lower key:value pairs, when explicitly specified, take precedence over the upper and the lower thresholds computed from the mean, the std, and the user specified threshold in the unit of standard deviation.</p>

<p>upper: </p>

<pre><code>    mean + std * threshold specified by user option -c
</code></pre>

<p>lower: </p>

<pre><code>    mean - std * threshold specified by user option -c
</code></pre>

<h3 id="option-descriptions">OPTION DESCRIPTIONS</h3>
<p>The -c option by default equals to 3. Therefore, the upper and the lower separation threshold would be: mean + 3 std and mean - 3 std respectively. It is useful to explicitly specify the upper and the lower separation thresholds when the insert size distribution is not symmetric to the mean. </p>

<p>The -o option enables per-chromosome/reference analysis and is much faster when the input files are in the bam format. Please index the bam file using “samtools index” to utilize this option. You need to specify the exact reference names as they are in the bam files. </p>

<p>When -e is on, BreakDancerMax tries to estimate the mean and the standard deviation insert size from the data instead of relying on user’s spec in the configuration file. Current implementation of this estimation process is slow. So it is recommended that users can specify the accurate thresholds in the configuration file. </p>

<p>The -l option tell BreakDancerMax that the data is produced from Illumina long insert circularized library </p>

<p>The -f option uses the Fisher’s methods to summarize scores from multiple libraries. It is recommended when there are many libraries. It ensures that the scores are independent of the number of libraries (uniform distribution of the P values) </p>

<p>The -q specifies the MAQ mapping quality threshold and can be used to skip reads that are not confidently mapped. </p>

<p>The -s specifies the minimal required size of a SV anchoring region from which the anomalously mapped reads are found. This parameter has some small effects on the SV detection accuracy. Increasing -s improves the specificity but also reduces the sensitivity. The default 7 bp seemed to work well. </p>

<p>The -b parameter specifies the number of anomalous regions resides in the RAM before SV hypotheses begin to form among these regions. The default works well in general. For dataset that is exceptionally large, it may be helpful to reduce it to cut the resident RAM usage. </p>

<p>The -d specifies a fastq file where all SV supporting reads will be saved in the fastq format. These reads can be realigned by other aligners such as novoalign, and then reanalyzed by BreakDancer.</p>



          </div>
            
            
        </div>

        <div id="content_sidebar">
          
  <div id="rounded_bg">
    <div align="center">
      <a href="install.html" class="btn_install">Install</a> <br/>

      
        <a href="https://github.com/genome/breakdancer" class="btn_view_on_github">View on GitHub</a> <br/>
      

      
        <a href="http://biostars.org/t/BreakDancer" class="btn_get_help">Get Help</a> <br />
      
    </div>
  </div>
  <br/>







        </div>

      </div>

      <div id="footer">
        <p>Genome Modeling Tools is produced by <a href="http://genome.wustl.edu/">The Genome Institute at Washington University School of Medicine</a></p>

      </div>
    </div>
  </body>
</html>
