<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
  
  

  <head>
    <title>Genome MuSiC (0.4)</title>
    <link rel="shortcut icon" href="/gmt_favicon.png" />
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="content-language" content="en" />
<link rel="stylesheet" href="/res/css/reset.css?20150720140317" media="screen,print"/>
<link rel="stylesheet" href="/res/css/main_sub.css?20150720140317" media="screen,print"/>

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="/res/js/jquery.currentPage.js"></script>
<script type="text/javascript" src="/res/js/jquery-ui.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
        $("#menu_main a").currentPage({
            indexPage: "index.html",
            anyPath: true
        });
    });
</script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24132169-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


  </head>
  <body>
    <div id="container">
      <div id="header">
          <div id="menu_top">
              <a href="/index.html" alt="GMT Main Site" id="gmt_link"> </a>
          </div>

          <div id="title">
              <div id="icon">
                  
                    <img src="/packages/genome-music/res/images/icon_48.png" width="48" height="48" alt="Genome MuSiC"/>
                  
              </div>
              <div id="site_title">
                  <h1>Genome MuSiC <span class="version">0.4</span></h1>
              </div>
          </div>

          <div id="menu_main">
              <ul>
                  
                  <li><a href="index.html">Overview</a></li>
                  
                  <li><a href="documentation.html">Documentation</a></li>
                  
                  <li><a href="install.html">Install</a></li>
                  
              </ul>
          </div>
      </div>
      <div id="content">

        <div id="helix_bottom">
          <img src="/res/images/helix_bottom.png" width="230" height="49" alt="helix_bottom" />
        </div>

        <div id="content_main">

          <h2></h2>
          <div class="rounded_bg">
              <div class="doc_nav_path">
  <p>
  <a href="#">genome</a> ::
  <a href="/packages/genome-music/documentation.html">music</a> ::
  <span class="last">survival</span>
  </p>
</div>
<h1><a name="___top"></a>
<p>genome music survival</p>
</h1>

<ul>
<li><a href="#___sec0">NAME</a></li>
<li><a href="#___sec1">VERSION</a></li>
<li><a href="#___sec2">SYNOPSIS</a></li>
<li><a href="#___sec3">REQUIRED ARGUMENTS</a></li>
<li><a href="#___sec4">OPTIONAL ARGUMENTS</a></li>
<li><a href="#___sec5">DESCRIPTION</a></li>
<li><a href="#___sec6">ARGUMENTS</a></li>
<li><a href="#___sec7">LICENSE</a></li>
<li><a href="#___sec8">AUTHORS</a></li>
<li><a href="#___sec9">SEE ALSO</a></li></ul>

<h1><a name="___sec0" href="#___top">NAME</a></h1>

<p>genome music survival - Create survival plots and P-values for clinical and mutational phenotypes.</p>
<br/>
<h1><a name="___sec1" href="#___top">VERSION</a></h1>

<p>This document describes genome music survival version 0.0401 (2014-02-13 at 08:01:08)</p>
<br/>
<h1><a name="___sec2" href="#___top">SYNOPSIS</a></h1>

<p>genome music survival --bam-list=?
--output-dir=?
[--maf-file=?] [--skip-silent] [--genetic-data-type=?] [--numeric-clinical-data-file=?] [--categorical-clinical-data-file=?] [--glm-clinical-data-file=?] [--phenotypes-to-include=?] [--legend-placement=?] [--skip-non-coding]</p>

<pre> ... music survival \
        --bam-list /path/myBamList.tsv \
        --maf-file /path/myMAF.tsv \
        --numeric-clinical-data-file /path/myNumericData.tsv \
        --categorical-clinical-data-file /path/myClassData.tsv \
        --output-dir /path/output_directory

 ... music survival \
        --bam-list /path/myBamList.tsv \
        --maf-file /path/myMAF.tsv \
        --glm-clinical-data-file /path/myGLMClinicalData.tsv \
        --output-dir /path/output_directory

 ... music survival \
        --bam-list /path/myBamList.tsv \
        --maf-file /path/myMAF.tsv \
        --genetic-data-type &#39;gene&#39; \
        --glm-clinical-data-file /path/myGlmClinicalData.tsv \
        --phenotypes-to-include &#39;Race,Gender,TP53&#39; \
        --output-dir /path/output_directory</pre>
<br/>
<h1><a name="___sec3" href="#___top">REQUIRED ARGUMENTS</a></h1>

<dl>
<dt><a name="bam-list_Text"
>bam-list <i>Text</i></a></dt>

<dd>
<p>List of sample names to be included in the analysis.
(See Description)</p>

<dt><a name="output-dir_Text"
>output-dir <i>Text</i></a></dt>

<dd>
<p>Directory where output files will be written</p>
</dd>
</dl>
<br/>
<h1><a name="___sec4" href="#___top">OPTIONAL ARGUMENTS</a></h1>

<dl>
<dt><a name="maf-file_Text"
>maf-file <i>Text</i></a></dt>

<dd>
<p>List of mutations in MAF format</p>

<dt><a name="skip-silent_Boolean"
>skip-silent <i>Boolean</i></a></dt>

<dd>
<p>Skip silent mutations from the provided MAF file</p>

<p>Default value &#39;true&#39; if not specified</p>

<dt><a name="noskip-silent_Boolean"
>noskip-silent <i>Boolean</i></a></dt>

<dd>
<p>Make skip-silent &#39;false&#39;</p>

<dt><a name="genetic-data-type_Text"
>genetic-data-type <i>Text</i></a></dt>

<dd>
<p>Correlate clinical data to &#34;gene&#34; or &#34;variant&#34; level data</p>

<p>Default value &#39;gene&#39; if not specified</p>

<dt><a name="numeric-clinical-data-file_Text"
>numeric-clinical-data-file <i>Text</i></a></dt>

<dd>
<p>Table of samples (y) vs.
numeric clinical data category (x)</p>

<dt><a name="categorical-clinical-data-file_Text"
>categorical-clinical-data-file <i>Text</i></a></dt>

<dd>
<p>Table of samples (y) vs.
categorical clinical data category (x)</p>

<dt><a name="glm-clinical-data-file_Text"
>glm-clinical-data-file <i>Text</i></a></dt>

<dd>
<p>Clinical traits,
mutational profiles,
other mixed clinical data (See DESCRIPTION).</p>

<dt><a name="phenotypes-to-include_Text"
>phenotypes-to-include <i>Text</i></a></dt>

<dd>
<p>Include only these genes and/or phenotypes in the anlaysis.
(COMMA-DELIMITED)</p>

<dt><a name="legend-placement_Text"
>legend-placement <i>Text</i></a></dt>

<dd>
<p>Choose one of &#39;bottomleft&#39;,
&#39;topleft&#39;,
&#39;topright&#39;,
or &#39;bottomright&#39;.</p>

<p>Default value &#39;bottomleft&#39; if not specified</p>

<dt><a name="skip-non-coding_Boolean"
>skip-non-coding <i>Boolean</i></a></dt>

<dd>
<p>Skip non-coding mutations from the provided MAF file</p>

<p>Default value &#39;true&#39; if not specified</p>

<dt><a name="noskip-non-coding_Boolean"
>noskip-non-coding <i>Boolean</i></a></dt>

<dd>
<p>Make skip-non-coding &#39;false&#39;</p>
</dd>
</dl>
<br/>
<h1><a name="___sec5" href="#___top">DESCRIPTION</a></h1>

<p>This command performs survival analysis and plots survival curves for mutational data,
as well as any clinical traits of interest as specified via the --phenotypes-to-include input parameter.
The analyses performed include the Kaplan-Meier estimator followed by the Cox Proportional Hazards model.
Outputs for each gene/clinical trait analyzed include survival curves,
a hazard ratio (with confidence intervals),
and P-values and FDRs describing the significance of the difference between survivors and non-survivors.</p>

<p>All clinical data files are searched for the required (case insensitive) &#34;vital_status&#34; and &#34;days_to_last_followup&#34; columns which are paired to phenotypes via sample IDs for the survival analysis.
The first column of all clinical data files MUST contain the sample IDs,
same as in other MuSiC tools.
By default,
analysis is performed on every gene present in the MAF.
Optionally,
the analysis may be limited to only specific genes by listing them (comma delimited) after the --phenotypes-to-include input parameter.
Survival analysis may also be performed on other columns in the clinical data file by adding the column headers to the list of entries specified after the --phenotypes-to-include input parameter.</p>

<p>Here are some general guildelines for creating clinical data input files:</p>

<ul>
<li>Headers are required.</li>

<li>The first column of each clinical data file must contain sample IDs which match those in both the --bam-list and the MAF variant list (in the MAF,
this is the Tumor_Sample_Barcode column,
specifically).</li>

<li>In at least one of the clinical data files input,
columns with headers &#34;vital_status&#34; and &#34;days_to_last_followup&#34; (case insensitive) must exist.
&#34;vital_status&#34; must be delineated by 1&#39;s and 0&#39;s,
where 0 denotes &#39;living&#39;,
and 1 denotes &#39;deceased&#39;.</li>
</ul>

<p>Note that all input files must be tab-separated.</p>
<br/>
<h1><a name="___sec6" href="#___top">ARGUMENTS</a></h1>

<dl>
<dt><a name="--bam-list"
>--bam-list</a></dt>

<dd>
<dl>
<dt><a 
>Provide a file containing sample names and normal/tumor BAM locations for each.
Use the tab- delimited format [sample_name normal_bam tumor_bam] per line.
This tool only needs sample_name,
so all other columns can be skipped.
The sample_name must be the same as the tumor sample names used in the MAF file (16th column,
with the header Tumor_Sample_Barcode).</a></dt>
</dl>
</dd>
</dl>
<br/>
<h1><a name="___sec7" href="#___top">LICENSE</a></h1>

<p>Copyright (C) 2010-2011 Washington University in St.
Louis.</p>

<p>It is released under the Lesser GNU Public License (LGPL) version 3.
See the associated LICENSE file in this distribution.</p>
<br/>
<h1><a name="___sec8" href="#___top">AUTHORS</a></h1>

<pre> Nathan D. Dees, Ph.D.
 Qunyuan Zhang, Ph.D.</pre>
<br/>
<h1><a name="___sec9" href="#___top">SEE ALSO</a></h1>

<p><b>genome-music</b>(1),
<b>genome</b>(1)</p>
<br/>

          </div>
            
            
        </div>

        <div id="content_sidebar">
          
  <div id="rounded_bg">
    <div align="center">
      <a href="install.html" class="btn_install">Install</a> <br/>

      

      
        <a href="http://biostars.org/t/MuSiC" class="btn_get_help">Get Help</a> <br />
      
    </div>
  </div>
  <br/>



  <div class="sidebar_menu">
    <h3>Versions</h3>
    <hr/>
    <ul>
    
      <li>
      <a href="install.html">0.4</a>
      </li>
    
    </ul>
  </div>
  <br/>





        </div>

      </div>

      <div id="footer">
        <p>Genome Modeling Tools is produced by <a href="http://genome.wustl.edu/">The Genome Institute at Washington University School of Medicine</a></p>

      </div>
    </div>
  </body>
</html>
