<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
  
  

  <head>
    <title>Genome MuSiC (0.4)</title>
    <link rel="shortcut icon" href="/gmt_favicon.png" />
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="content-language" content="en" />
<link rel="stylesheet" href="/res/css/reset.css?20140813154312" media="screen,print"/>
<link rel="stylesheet" href="/res/css/main_sub.css?20140813154312" media="screen,print"/>

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="/res/js/jquery.currentPage.js"></script>
<script type="text/javascript" src="/res/js/jquery-ui.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
        $("#menu_main a").currentPage({
            indexPage: "index.html",
            anyPath: true
        });
    });
</script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24132169-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


  </head>
  <body>
    <div id="container">
      <div id="header">
          <div id="menu_top">
              <a href="/index.html" alt="GMT Main Site" id="gmt_link"> </a>
          </div>

          <div id="title">
              <div id="icon">
                  
                    <img src="/packages/genome-music/res/images/icon_48.png" width="48" height="48" alt="Genome MuSiC"/>
                  
              </div>
              <div id="site_title">
                  <h1>Genome MuSiC <span class="version">0.4</span></h1>
              </div>
          </div>

          <div id="menu_main">
              <ul>
                  
                  <li><a href="index.html">Overview</a></li>
                  
                  <li><a href="documentation.html">Documentation</a></li>
                  
                  <li><a href="install.html">Install</a></li>
                  
              </ul>
          </div>
      </div>
      <div id="content">

        <div id="helix_bottom">
          <img src="/res/images/helix_bottom.png" width="230" height="49" alt="helix_bottom" />
        </div>

        <div id="content_main">

          <h2></h2>
          <div class="rounded_bg">
              <div class="doc_nav_path">
  <p>
  <a href="#">genome</a> ::
  <a href="/packages/genome-music/documentation.html">music</a> ::
  <span class="last">clinical-correlation</span>
  </p>
</div>
<h1><a name="___top"></a>
<p>genome music clinical-correlation</p>
</h1>

<ul>
<li><a href="#___sec0">NAME</a></li>
<li><a href="#___sec1">VERSION</a></li>
<li><a href="#___sec2">SYNOPSIS</a></li>
<li><a href="#___sec3">REQUIRED ARGUMENTS</a></li>
<li><a href="#___sec4">OPTIONAL ARGUMENTS</a></li>
<li><a href="#___sec5">DESCRIPTION</a></li>
<li><a href="#___sec6">ARGUMENTS</a></li>
<li><a href="#___sec7">LICENSE</a></li>
<li><a href="#___sec8">AUTHORS</a></li>
<li><a href="#___sec9">SEE ALSO</a></li></ul>

<h1><a name="___sec0" href="#___top">NAME</a></h1>

<p>genome music clinical-correlation - Correlate phenotypic traits against mutated genes,
or against individual variants</p>
<br/>
<h1><a name="___sec1" href="#___top">VERSION</a></h1>

<p>This document describes genome music clinical-correlation version 0.0401 (2014-02-13 at 08:01:08)</p>
<br/>
<h1><a name="___sec2" href="#___top">SYNOPSIS</a></h1>

<p>genome music clinical-correlation --bam-list=?
--output-file=?
[--maf-file=?] [--glm-clinical-data-file=?] [--use-maf-in-glm] [--skip-non-coding] [--skip-silent] [--clinical-correlation-matrix-file=?] [--input-clinical-correlation-matrix-file=?] [--genetic-data-type=?] [--numeric-clinical-data-file=?] [--numerical-data-test-method=?] [--categorical-clinical-data-file=?] [--glm-model-file=?]</p>

<pre> ... music clinical-correlation \
        --bam-list /path/myBamList.tsv \
        --maf-file /path/myMAF.tsv \
        --numeric-clinical-data-file /path/myNumericData.tsv \
        --genetic-data-type &#39;gene&#39; \
        --output-file /path/output_file

 ... music clinical-correlation \
        --maf-file /path/myMAF.tsv \
        --bam-list /path/myBamList.tsv \
        --numeric-clinical-data-file /path/myNumericData.tsv \
        --categorical-clinical-data-file /path/myClassData.tsv \
        --genetic-data-type &#39;gene&#39; \
        --output-file /path/output_file

 ... music clinical-correlation \
        --maf-file /path/myMAF.tsv \
        --bam-list /path/myBamList.tsv \
        --output-file /path/output_file \
        --glm-model-file /path/model.tsv \
        --glm-clinical-data-file /path/glm_clinical_data.tsv \
        --use-maf-in-glm</pre>
<br/>
<h1><a name="___sec3" href="#___top">REQUIRED ARGUMENTS</a></h1>

<dl>
<dt><a name="bam-list_Text"
>bam-list <i>Text</i></a></dt>

<dd>
<p>Tab delimited list of BAM files [sample_name,
normal_bam,
tumor_bam] (See Description)</p>

<dt><a name="output-file_Text"
>output-file <i>Text</i></a></dt>

<dd>
<p>Results of clinical-correlation tool.
Will have suffix added for data type</p>
</dd>
</dl>
<br/>
<h1><a name="___sec4" href="#___top">OPTIONAL ARGUMENTS</a></h1>

<dl>
<dt><a name="maf-file_Text"
>maf-file <i>Text</i></a></dt>

<dd>
<p>List of mutations using TCGA MAF specification v2.3</p>

<dt><a name="glm-clinical-data-file_Text"
>glm-clinical-data-file <i>Text</i></a></dt>

<dd>
<p>Clinical traits,
mutational profiles,
other mixed clinical data (See DESCRIPTION)</p>

<dt><a name="use-maf-in-glm_Boolean"
>use-maf-in-glm <i>Boolean</i></a></dt>

<dd>
<p>Create a variant matrix from the MAF file as variant input to GLM analysis.</p>

<p>Default value &#39;false&#39; (--nouse-maf-in-glm) if not specified</p>

<dt><a name="nouse-maf-in-glm_Boolean"
>nouse-maf-in-glm <i>Boolean</i></a></dt>

<dd>
<p>Make use-maf-in-glm &#39;false&#39;</p>

<dt><a name="skip-non-coding_Boolean"
>skip-non-coding <i>Boolean</i></a></dt>

<dd>
<p>Skip non-coding mutations from the provided MAF file</p>

<p>Default value &#39;true&#39; if not specified</p>

<dt><a name="noskip-non-coding_Boolean"
>noskip-non-coding <i>Boolean</i></a></dt>

<dd>
<p>Make skip-non-coding &#39;false&#39;</p>

<dt><a name="skip-silent_Boolean"
>skip-silent <i>Boolean</i></a></dt>

<dd>
<p>Skip silent mutations from the provided MAF file</p>

<p>Default value &#39;true&#39; if not specified</p>

<dt><a name="noskip-silent_Boolean"
>noskip-silent <i>Boolean</i></a></dt>

<dd>
<p>Make skip-silent &#39;false&#39;</p>

<dt><a name="clinical-correlation-matrix-file_Text"
>clinical-correlation-matrix-file <i>Text</i></a></dt>

<dd>
<p>Specify a file to store the sample-vs-gene matrix created during calculations</p>

<dt><a name="input-clinical-correlation-matrix-file_Text"
>input-clinical-correlation-matrix-file <i>Text</i></a></dt>

<dd>
<p>Instead of creating this from the MAF,
input the sample-vs-gene matrix for calculations</p>

<dt><a name="genetic-data-type_Text"
>genetic-data-type <i>Text</i></a></dt>

<dd>
<p>Correlate clinical data to &#34;gene&#34; or &#34;variant&#34; level data</p>

<p>Default value &#39;gene&#39; if not specified</p>

<dt><a name="numeric-clinical-data-file_Text"
>numeric-clinical-data-file <i>Text</i></a></dt>

<dd>
<p>Table of samples (y) vs.
numeric clinical data category (x)</p>

<dt><a name="numerical-data-test-method_Text"
>numerical-data-test-method <i>Text</i></a></dt>

<dd>
<p>Either &#39;cor&#39; for Pearson Correlation or &#39;wilcox&#39; for the Wilcoxon Rank-Sum Test for numerical clinical data</p>

<p>Default value &#39;cor&#39; if not specified</p>

<dt><a name="categorical-clinical-data-file_Text"
>categorical-clinical-data-file <i>Text</i></a></dt>

<dd>
<p>Table of samples (y) vs.
categorical clinical data category (x)</p>

<dt><a name="glm-model-file_Text"
>glm-model-file <i>Text</i></a></dt>

<dd>
<p>File outlining the type of model,
response variable,
covariants,
etc.
for the GLM analysis.
(See DESCRIPTION)</p>
</dd>
</dl>
<br/>
<h1><a name="___sec5" href="#___top">DESCRIPTION</a></h1>

<p>This command relates clinical traits and mutational data.
Either one can perform correlation analysis between mutations recorded in a MAF and the particular phenotypic traits recorded in clinical data files for the same samples,
or one can run a generalized linear model (GLM) analysis on the same types of data.</p>

<p>The clinical data files for correlation must be separated between numeric and categoric data and must follow these conventions:</p>

<ul>
<li>Headers are required</li>

<li>Each file must include at least 1 sample_id column and 1 attribute column,
with the format being [sample_id clinical_data_attribute_1 clinical_data_attribute_2 ...]</li>

<li>The sample ID must match the sample ID listed in the MAF under &#34;Tumor_Sample_Barcode&#34; for relating the mutations of this sample.</li>
</ul>

<p>Note the importance of the headers: the header for each clinical_data_attribute will appear in the output file to denote relationships with the mutation data from the MAF.</p>

<p>Internally,
the input data is fed into an R script which calculates a P-value representing the probability that the correlation seen between the mutations in each gene (or variant) and each phenotype trait are random.
Lower P-values indicate lower randomness,
or likely true correlations.</p>

<p>The results are saved to the output filename given with a suffix appended; &#34;.numeric.csv&#34; will be appended for results derived from numeric clinical data,
and &#34;.categorical.csv&#34; will be appended for results derived from categorical clinical data.
Also,
&#34;.glm.csv&#34; will be appended to the output filename for GLM results.</p>

<p>The GLM analysis accepts a mixed numeric and categoric clinical data file,
input using the parameter --glm-clinical-data-file.
GLM clinical data must adhere to the formats described above for the correlation clinical data files.
GLM also requires the user to input a --glm-model-file.
This file requires specific headers and defines the analysis to be performed rather exactly.
Here are the conventions required for this file:</p>

<ul>
<li>Columns must be ordered as such:</li>

<li>[ analysis_type clinical_data_trait_name variant/gene_name covariates memo ]</li>

<li>The &#39;analysis_type&#39; column must contain either &#34;Q&#34;,
indicating a quantative trait,
or &#34;B&#34;,
indicating a binary trait will be examined.</li>

<li>The &#39;clinical_data_trait_name&#39; is the name of a clinical data trait defined by being a header in the --glm-clinical-data-file.</li>

<li>The &#39;variant/gene_name&#39; can either be the name of one or more columns from the --glm-clinical-data-file,
or the name of one or more mutated gene names from the MAF,
separated by &#34;|&#34;.
If this column is left blank,
or instead contains &#34;NA&#34;,
then each column from either the variant mutation matrix (--use-maf-in-glm) or alternatively the --glm-clinical-data-file is used consecutively as the variant column in independent analyses.</li>

<li>&#39;covariates&#39; are the names of one or more columns from the --glm-clinical-data-file,
separated by &#34;+&#34;.</li>

<li>&#39;memo&#39; is any note deemed useful to the user.
It will be printed in the output data file for reference.</li>
</ul>

<p>GLM analysis may be performed using solely the data input into --glm-clinical-data-file,
as described above,
or alternatively,
mutational data from the MAF may be included as variants in the GLM analysis,
as also described above.
Use the --use-maf-in-glm flag to include the mutation matrix derived from the maf as variant data.</p>

<p>Note that all input files for both correlation and GLM analysis must be tab-separated.</p>

<h1><a name="___sec6" href="#___top">ARGUMENTS</a></h1>

<dl>
<dt><a name="--bam-list"
>--bam-list</a></dt>

<dd>
<dl>
<dt><a 
>Provide a file containing sample names and normal/tumor BAM locations for each.
Use the tab- delimited format [sample_name normal_bam tumor_bam] per line.
This tool only needs sample_name,
so all other columns can be skipped.
The sample_name must be the same as the tumor sample names used in the MAF file (16th column,
with the header Tumor_Sample_Barcode).</a></dt>
</dl>
</dd>
</dl>
<br/>
<h1><a name="___sec7" href="#___top">LICENSE</a></h1>

<p>Copyright (C) 2010-2011 Washington University in St.
Louis.</p>

<p>It is released under the Lesser GNU Public License (LGPL) version 3.
See the associated LICENSE file in this distribution.</p>
<br/>
<h1><a name="___sec8" href="#___top">AUTHORS</a></h1>

<pre> Nathan D. Dees, Ph.D.
 Qunyuan Zhang, Ph.D.
 William Schierding, M.S.</pre>
<br/>
<h1><a name="___sec9" href="#___top">SEE ALSO</a></h1>

<p><b>genome-music</b>(1),
<b>genome</b>(1)</p>
<br/>

          </div>
            
            
        </div>

        <div id="content_sidebar">
          
  <div id="rounded_bg">
    <div align="center">
      <a href="install.html" class="btn_install">Install</a> <br/>

      

      
        <a href="http://biostars.org/t/MuSiC" class="btn_get_help">Get Help</a> <br />
      
    </div>
  </div>
  <br/>



  <div class="sidebar_menu">
    <h3>Versions</h3>
    <hr/>
    <ul>
    
      <li>
      <a href="install.html">0.4</a>
      </li>
    
    </ul>
  </div>
  <br/>





        </div>

      </div>

      <div id="footer">
        <p>Genome Modeling Tools is produced by <a href="http://genome.wustl.edu/">The Genome Institute at Washington University School of Medicine</a></p>

      </div>
    </div>
  </body>
</html>
