<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
  
  

  <head>
    <title>Genome MuSiC (0.4)</title>
    <link rel="shortcut icon" href="/gmt_favicon.png" />
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="content-language" content="en" />
<link rel="stylesheet" href="/res/css/reset.css?20140813154312" media="screen,print"/>
<link rel="stylesheet" href="/res/css/main_sub.css?20140813154312" media="screen,print"/>

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="/res/js/jquery.currentPage.js"></script>
<script type="text/javascript" src="/res/js/jquery-ui.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
        $("#menu_main a").currentPage({
            indexPage: "index.html",
            anyPath: true
        });
    });
</script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24132169-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


  </head>
  <body>
    <div id="container">
      <div id="header">
          <div id="menu_top">
              <a href="/index.html" alt="GMT Main Site" id="gmt_link"> </a>
          </div>

          <div id="title">
              <div id="icon">
                  
                    <img src="/packages/genome-music/res/images/icon_48.png" width="48" height="48" alt="Genome MuSiC"/>
                  
              </div>
              <div id="site_title">
                  <h1>Genome MuSiC <span class="version">0.4</span></h1>
              </div>
          </div>

          <div id="menu_main">
              <ul>
                  
                  <li><a href="index.html">Overview</a></li>
                  
                  <li><a href="documentation.html">Documentation</a></li>
                  
                  <li><a href="install.html">Install</a></li>
                  
              </ul>
          </div>
      </div>
      <div id="content">

        <div id="helix_bottom">
          <img src="/res/images/helix_bottom.png" width="230" height="49" alt="helix_bottom" />
        </div>

        <div id="content_main">

          <h2></h2>
          <div class="rounded_bg">
              <div class="doc_nav_path">
  <p>
  <a href="#">genome</a> ::
  <a href="/packages/genome-music/documentation.html">music</a> ::
  <a href="/packages/genome-music/genome-music-bmr.html">bmr</a> ::
  <span class="last">calc-covg-helper</span>
  </p>
</div>
<h1><a name="___top"></a>
<p>genome music bmr calc-covg-helper</p>
</h1>

<ul>
<li><a href="#___sec0">NAME</a></li>
<li><a href="#___sec1">VERSION</a></li>
<li><a href="#___sec2">SYNOPSIS</a></li>
<li><a href="#___sec3">REQUIRED ARGUMENTS</a></li>
<li><a href="#___sec4">OPTIONAL ARGUMENTS</a></li>
<li><a href="#___sec5">DESCRIPTION</a></li>
<li><a href="#___sec6">ARGUMENTS</a></li>
<li><a href="#___sec7">LICENSE</a></li>
<li><a href="#___sec8">AUTHORS</a></li>
<li><a href="#___sec9">SEE ALSO</a></li></ul>

<h1><a name="___sec0" href="#___top">NAME</a></h1>

<p>genome music bmr calc-covg-helper - Uses calcRoiCovg.c to count covered bases per-gene for a tumor-normal pair of BAMs.</p>
<br/>
<h1><a name="___sec1" href="#___top">VERSION</a></h1>

<p>This document describes genome music bmr calc-covg-helper version 0.0401 (2014-02-13 at 08:01:08)</p>
<br/>
<h1><a name="___sec2" href="#___top">SYNOPSIS</a></h1>

<p>genome music bmr calc-covg-helper --roi-file=?
--reference-sequence=?
--normal-tumor-bam-pair=?
[--output-file=?] [--output-dir=?] [--normal-min-depth=?] [--tumor-min-depth=?] [--min-mapq=?]</p>

<p>General usage:</p>

<pre> ... music bmr calc-covg-helper \
    --normal-tumor-bam-pair &#34;sample-name path/to/normal_bam path/to/tumor_bam&#34; \
    --reference-sequence input_dir/all_sequences.fa \
    --output-file output_file \
    --roi-file input_dir/all_coding_exons.tsv</pre>
<br/>
<h1><a name="___sec3" href="#___top">REQUIRED ARGUMENTS</a></h1>

<dl>
<dt><a name="roi-file_Text"
>roi-file <i>Text</i></a></dt>

<dd>
<p>Tab delimited list of ROIs [chr start stop gene_name] (See Description)</p>

<dt><a name="reference-sequence_Text"
>reference-sequence <i>Text</i></a></dt>

<dd>
<p>Path to reference sequence in FASTA format</p>

<dt><a name="normal-tumor-bam-pair_Text"
>normal-tumor-bam-pair <i>Text</i></a></dt>

<dd>
<p>Tab delimited line with sample name,
path to normal bam file,
and path to tumor bam file (See Description)</p>
</dd>
</dl>
<br/>
<h1><a name="___sec4" href="#___top">OPTIONAL ARGUMENTS</a></h1>

<dl>
<dt><a name="output-file_Text"
>output-file <i>Text</i></a></dt>

<dd>
<p>Output file path.
Specify either output-file or output-directory.</p>

<dt><a name="output-dir_Text"
>output-dir <i>Text</i></a></dt>

<dd>
<p>Output directory path.
Specify either output-file or output-directory</p>

<dt><a name="normal-min-depth_Integer"
>normal-min-depth <i>Integer</i></a></dt>

<dd>
<p>The minimum read depth to consider a Normal BAM base as covered</p>

<p>Default value &#39;6&#39; if not specified</p>

<dt><a name="tumor-min-depth_Integer"
>tumor-min-depth <i>Integer</i></a></dt>

<dd>
<p>The minimum read depth to consider a Tumor BAM base as covered</p>

<p>Default value &#39;8&#39; if not specified</p>

<dt><a name="min-mapq_Integer"
>min-mapq <i>Integer</i></a></dt>

<dd>
<p>The minimum mapping quality of reads to consider towards read depth counts</p>

<p>Default value &#39;20&#39; if not specified</p>
</dd>
</dl>
<br/>
<h1><a name="___sec5" href="#___top">DESCRIPTION</a></h1>

<p>This script counts bases with sufficient coverage in the ROIs of each gene in the given pair of tumor-normal BAM files and categorizes them into - AT,
CG (non-CpG),
and CpG counts.
It also adds up these base-counts across all ROIs of each gene in the sample,
but covered bases that lie within overlapping ROIs are not counted more than once towards these total counts.</p>

<h1><a name="___sec6" href="#___top">ARGUMENTS</a></h1>

<dl>
<dt><a name="--roi-file"
>--roi-file</a></dt>

<dd>
<dl>
<dt><a 
>The regions of interest (ROIs) of each gene are typically regions targeted for sequencing or are merged exon loci (from multiple transcripts) of genes with 2-bp flanks (splice junctions).
ROIs from the same chromosome must be listed adjacent to each other in this file.
This allows the underlying C-based code to run much more efficiently and avoid re-counting bases seen in overlapping ROIs (for overall covered base counts).
For per-gene base counts,
an overlapping base will be counted each time it appears in an ROI of the same gene.
To avoid this,
be sure to merge together overlapping ROIs of the same gene.
BEDtools&#39; mergeBed can help if used per gene.</a></dt>
</dl>

<dt><a name="--reference-sequence"
>--reference-sequence</a></dt>

<dd>
<dl>
<dt><a 
>The reference sequence in FASTA format.
If a reference sequence index is not found next to this file (a .fai file),
it will be created.</a></dt>
</dl>

<dt><a name="--normal-tumor-bam-pair"
>--normal-tumor-bam-pair</a></dt>

<dd>
<dl>
<dt><a name="&#34;sample-name_path/to/normal_bam_path/to/tumor_bam&#34;"
>&#34;sample-name path/to/normal_bam path/to/tumor_bam&#34;</a></dt>
</dl>

<dt><a name="--output-file"
>--output-file</a></dt>

<dd>
<dl>
<dt><a name="Specify_an_output_file_where_the_per-ROI_covered_base_counts_will_be_written"
>Specify an output file where the per-ROI covered base counts will be written</a></dt>
</dl>
</dd>
</dl>

<h1><a class='u'
name="POD_ERRORS"
>POD ERRORS</a></h1>

<p>Hey!
<b>The above document had some coding errors,
which are explained below:</b></p>

<dl>
<dt><a name="Around_line_6:"
>Around line 6:</a></dt>

<dd>
<p>=over without closing =back</p>
</dd>
</dl>
<br/>
<h1><a name="___sec7" href="#___top">LICENSE</a></h1>

<p>Copyright (C) 2010-2011 Washington University in St.
Louis.</p>

<p>It is released under the Lesser GNU Public License (LGPL) version 3.
See the associated LICENSE file in this distribution.</p>
<br/>
<h1><a name="___sec8" href="#___top">AUTHORS</a></h1>

<pre> Cyriac Kandoth, Ph.D.</pre>
<br/>
<h1><a name="___sec9" href="#___top">SEE ALSO</a></h1>

<p><b>genome-music-bmr</b>(1),
<b>genome-music</b>(1),
<b>genome</b>(1)</p>
<br/>

          </div>
            
            
        </div>

        <div id="content_sidebar">
          
  <div id="rounded_bg">
    <div align="center">
      <a href="install.html" class="btn_install">Install</a> <br/>

      

      
        <a href="http://biostars.org/t/MuSiC" class="btn_get_help">Get Help</a> <br />
      
    </div>
  </div>
  <br/>



  <div class="sidebar_menu">
    <h3>Versions</h3>
    <hr/>
    <ul>
    
      <li>
      <a href="install.html">0.4</a>
      </li>
    
    </ul>
  </div>
  <br/>





        </div>

      </div>

      <div id="footer">
        <p>Genome Modeling Tools is produced by <a href="http://genome.wustl.edu/">The Genome Institute at Washington University School of Medicine</a></p>

      </div>
    </div>
  </body>
</html>
