<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
  
  

  <head>
    <title>Genome MuSiC (0.4)</title>
    <link rel="shortcut icon" href="/gmt_favicon.png" />
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="content-language" content="en" />
<link rel="stylesheet" href="/res/css/reset.css?20140813143935" media="screen,print"/>
<link rel="stylesheet" href="/res/css/main_sub.css?20140813143935" media="screen,print"/>

<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
<script type="text/javascript" src="/res/js/jquery.currentPage.js"></script>
<script type="text/javascript" src="/res/js/jquery-ui.js"></script>
<script type="text/javascript">
    $(document).ready(function() {
        $("#menu_main a").currentPage({
            indexPage: "index.html",
            anyPath: true
        });
    });
</script>

<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24132169-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>


  </head>
  <body>
    <div id="container">
      <div id="header">
          <div id="menu_top">
              <a href="/index.html" alt="GMT Main Site" id="gmt_link"> </a>
          </div>

          <div id="title">
              <div id="icon">
                  
                    <img src="/packages/genome-music/res/images/icon_48.png" width="48" height="48" alt="Genome MuSiC"/>
                  
              </div>
              <div id="site_title">
                  <h1>Genome MuSiC <span class="version">0.4</span></h1>
              </div>
          </div>

          <div id="menu_main">
              <ul>
                  
                  <li><a href="index.html">Overview</a></li>
                  
                  <li><a href="documentation.html">Documentation</a></li>
                  
                  <li><a href="install.html">Install</a></li>
                  
              </ul>
          </div>
      </div>
      <div id="content">

        <div id="helix_bottom">
          <img src="/res/images/helix_bottom.png" width="230" height="49" alt="helix_bottom" />
        </div>

        <div id="content_main">

          <h2></h2>
          <div class="rounded_bg">
              <div class="doc_nav_path">
  <p>
  <a href="#">genome</a> ::
  <a href="/packages/genome-music/documentation.html">music</a> ::
  <a href="/packages/genome-music/genome-music-bmr.html">bmr</a> ::
  <span class="last">calc-covg</span>
  </p>
</div>
<h1><a name="___top"></a>
<p>genome music bmr calc-covg</p>
</h1>

<ul>
<li><a href="#___sec0">NAME</a></li>
<li><a href="#___sec1">VERSION</a></li>
<li><a href="#___sec2">SYNOPSIS</a></li>
<li><a href="#___sec3">REQUIRED ARGUMENTS</a></li>
<li><a href="#___sec4">OPTIONAL ARGUMENTS</a></li>
<li><a href="#___sec5">DESCRIPTION</a></li>
<li><a href="#___sec6">ARGUMENTS</a></li>
<li><a href="#___sec7">LICENSE</a></li>
<li><a href="#___sec8">AUTHORS</a></li>
<li><a href="#___sec9">SEE ALSO</a></li></ul>

<h1><a name="___sec0" href="#___top">NAME</a></h1>

<p>genome music bmr calc-covg - Uses calcRoiCovg.c to count covered bases per-gene for each given tumor-normal pair of BAMs.</p>
<br/>
<h1><a name="___sec1" href="#___top">VERSION</a></h1>

<p>This document describes genome music bmr calc-covg version 0.0401 (2014-02-13 at 08:01:08)</p>
<br/>
<h1><a name="___sec2" href="#___top">SYNOPSIS</a></h1>

<p>genome music bmr calc-covg --gene-covg-dir=?
--roi-file=?
--reference-sequence=?
--bam-list=?
--output-dir=?
[--cmd-list-file=?] [--cmd-prefix=?] [--normal-min-depth=?] [--tumor-min-depth=?] [--min-mapq=?]</p>

<p>General usage:</p>

<pre> ... music bmr calc-covg \
    --bam-list input_dir/bam_list \
    --output-dir output_dir/ \
    --reference-sequence input_dir/all_sequences.fa \
    --roi-file input_dir/all_coding_exons.tsv</pre>

<p>To create a list of commands that will allow the processing of each tumor-normal pair in parallel with an LSF job scheduler:</p>

<pre> ... music bmr calc-covg \
    --bam-list input_dir/bam_list \
    --output-dir output_dir/ \
    --reference-sequence input_dir/all_sequences.fa \
    --roi-file input_dir/all_coding_exons.tsv \
    --cmd_list_file parallelizable_commands \
    --cmd_prefix bsub</pre>

<p>In the above case, the commands printed into the output file &#34;parallelizable_commands&#34; can be run in parallel. After they complete, rerun this script as printed directly below (--cmd_list_file and --cmd_prefix have been removed) to merge the parallelized calculations:</p>

<pre> ... music bmr calc-covg \
    --bam-list input_dir/bam_list \
    --output-dir output_dir/ \
    --reference-sequence input_dir/all_sequences.fa \
    --roi-file input_dir/all_coding_exons.tsv</pre>
<br/>
<h1><a name="___sec3" href="#___top">REQUIRED ARGUMENTS</a></h1>

<dl>
<dt><a name="gene-covg-dir_Text"
>gene-covg-dir <i>Text</i></a></dt>

<dd>
<p>Directory where per-sample gene coverage files are located</p>

<dt><a name="roi-file_Text"
>roi-file <i>Text</i></a></dt>

<dd>
<p>Tab delimited list of ROIs [chr start stop gene_name] (See Description)</p>

<dt><a name="reference-sequence_Text"
>reference-sequence <i>Text</i></a></dt>

<dd>
<p>Path to reference sequence in FASTA format</p>

<dt><a name="bam-list_Text"
>bam-list <i>Text</i></a></dt>

<dd>
<p>Tab delimited list of BAM files [sample_name normal_bam tumor_bam] (See Description)</p>

<dt><a name="output-dir_Text"
>output-dir <i>Text</i></a></dt>

<dd>
<p>Directory where output files and subdirectories will be written</p>
</dd>
</dl>
<br/>
<h1><a name="___sec4" href="#___top">OPTIONAL ARGUMENTS</a></h1>

<dl>
<dt><a name="cmd-list-file_Text"
>cmd-list-file <i>Text</i></a></dt>

<dd>
<p>A file to write calcRoiCovg commands to (See Description)</p>

<dt><a name="cmd-prefix_Text"
>cmd-prefix <i>Text</i></a></dt>

<dd>
<p>A command that submits a job to your cluster (See Description)</p>

<dt><a name="normal-min-depth_Integer"
>normal-min-depth <i>Integer</i></a></dt>

<dd>
<p>The minimum read depth to consider a Normal BAM base as covered</p>

<dt><a name="tumor-min-depth_Integer"
>tumor-min-depth <i>Integer</i></a></dt>

<dd>
<p>The minimum read depth to consider a Tumor BAM base as covered</p>

<dt><a name="min-mapq_Integer"
>min-mapq <i>Integer</i></a></dt>

<dd>
<p>The minimum mapping quality of reads to consider towards read depth counts</p>
</dd>
</dl>
<br/>
<h1><a name="___sec5" href="#___top">DESCRIPTION</a></h1>

<p>This script counts bases with sufficient coverage in the ROIs of each gene in the given pairs of tumor-normal BAM files and categorizes them into - AT,
CG (non-CpG),
and CpG counts.
It also adds up these base-counts across all ROIs of each gene for each sample,
but covered bases that lie within overlapping ROIs are not counted more than once towards these total counts.</p>

<p>By default,
this script runs a C-based tool named calcRoiCovg for each sample one after another,
taking ~30 mins per sample to generate per-ROI covered base counts.
If the results of calcRoiCovg for a sample already exists in the output subdirectory roi_covgs,
re-calculation is skipped.
This allows you to run your own calcRoiCovg jobs in parallel or on multiple machines (Keep reading).</p>

<p>Speed things up by running calcRoiCovg jobs in parallel: If a compute cluster or multiple machines are available,
run this script twice as follows:</p>

<ul>
<li>Define cmd-list-file and cmd-prefix to generate a file with commands that can be submitted to a cluster or run manually.
These jobs will write per-ROI base counts in a subdirectory roi_covgs.</li>

<li>After all the parallelized calcRoiCovg jobs are completed,
run this script again to add them up and generate the final per-gene base counts in a subdirectory gene_covgs.
Remember to remove the cmd-list-file and cmd-prefix arguments or you will just be re-creating a list of commands.</li>
</ul>
<br/>
<h1><a name="___sec6" href="#___top">ARGUMENTS</a></h1>

<dl>
<dt><a name="--roi-file"
>--roi-file</a></dt>

<dd>
<dl>
<dt><a 
>The regions of interest (ROIs) of each gene are typically regions targeted for sequencing or are merged exon loci (from multiple transcripts) of genes with 2-bp flanks (splice junctions).
ROIs from the same chromosome must be listed adjacent to each other in this file.
This allows the underlying C-based code to run much more efficiently and avoid re-counting bases seen in overlapping ROIs (for overall covered base counts).
For per-gene base counts,
an overlapping base will be counted each time it appears in an ROI of the same gene.
To avoid this,
be sure to merge together overlapping ROIs of the same gene.
BEDtools&#39; mergeBed can help if used per gene.</a></dt>
</dl>

<dt><a name="--reference-sequence"
>--reference-sequence</a></dt>

<dd>
<dl>
<dt><a 
>The reference sequence in FASTA format.
If a reference sequence index is not found next to this file (a .fai file),
it will be created.</a></dt>
</dl>

<dt><a name="--bam-list"
>--bam-list</a></dt>

<dd>
<dl>
<dt><a 
>Provide a file containing sample names and normal/tumor BAM locations for each.
Use the tab- delimited format [sample_name normal_bam tumor_bam] per line.
Additional columns like clinical data are allowed,
but ignored.
The sample_name must be the same as the tumor sample names used in the MAF file (16th column,
with the header Tumor_Sample_Barcode).</a></dt>
</dl>

<dt><a name="--output-dir"
>--output-dir</a></dt>

<dd>
<dl>
<dt><a 
>Specify an output directory where the following will be created/written: roi_covgs: Subdirectory containing per-ROI covered base counts for each sample.
gene_covgs: Subdirectory containing per-gene covered base counts for each sample.
total_covgs: File containing the overall non-overlapping coverages per sample.</a></dt>
</dl>

<dt><a name="--cmd-list-file"
>--cmd-list-file</a></dt>

<dd>
<dl>
<dt><a 
>Specify a file into which a list of calcRoiCovg jobs will be written to.
These can be scheduled in parallel,
and will write per-ROI covered base-counts into the output subdirectory roi_covgs.
If cmd-list-file is left unspecified,
this script runs calcRoiCovg per sample one after another,
taking ~30 mins per sample,
but it skips samples whose output is already in roi_covgs.</a></dt>
</dl>

<dt><a name="--cmd-prefix"
>--cmd-prefix</a></dt>

<dd>
<dl>
<dt><a 
>Specify a job submission command that will be prefixed to each command in cmd-list-file.
This makes batch submission easier.
Just run the cmd-list-file file as a shell script to submit jobs.
cmd-prefix is &#34;bsub&#34; if your cluster uses the LSF job scheduler,
or &#34;qsub&#34; in Torque.
Add arguments as necessary.
For example,
&#34;bsub -M 4GB&#34; sets a soft memory limit of 4GB.</a></dt>
</dl>
</dd>
</dl>
<br/>
<h1><a name="___sec7" href="#___top">LICENSE</a></h1>

<p>Copyright (C) 2010-2011 Washington University in St.
Louis.</p>

<p>It is released under the Lesser GNU Public License (LGPL) version 3.
See the associated LICENSE file in this distribution.</p>
<br/>
<h1><a name="___sec8" href="#___top">AUTHORS</a></h1>

<pre> Cyriac Kandoth, Ph.D.</pre>
<br/>
<h1><a name="___sec9" href="#___top">SEE ALSO</a></h1>

<p><b>genome-music-bmr</b>(1),
<b>genome-music</b>(1),
<b>genome</b>(1)</p>
<br/>

          </div>
            
            
        </div>

        <div id="content_sidebar">
          
  <div id="rounded_bg">
    <div align="center">
      <a href="install.html" class="btn_install">Install</a> <br/>

      

      
        <a href="http://biostars.org/t/MuSiC" class="btn_get_help">Get Help</a> <br />
      
    </div>
  </div>
  <br/>



  <div class="sidebar_menu">
    <h3>Versions</h3>
    <hr/>
    <ul>
    
      <li>
      <a href="install.html">0.4</a>
      </li>
    
    </ul>
  </div>
  <br/>





        </div>

      </div>

      <div id="footer">
        <p>Genome Modeling Tools is produced by <a href="http://genome.wustl.edu/">The Genome Institute at Washington University School of Medicine</a></p>

      </div>
    </div>
  </body>
</html>
